src/components/QrcodeStream.vue

```diff
@@ -261,33 +264,37 @@ const onLocate = (detectedCodes: DetectedBarcode[]) => {
     props.track(adjustedCodes, ctx)
   }
 }
-</script>
 
-<style lang="css" scoped>
-.qrcode-stream-wrapper {
-  width: 100%;
-  height: 100%;
-
-  position: relative;
-  z-index: 0;
-}
+/**
+ * Styling is all inline to avoid generating an external style.css file.
+ * Component users shouldn't have to figure out how to setup their bundler to 
+ * import external CSS.
+ */
 
-.qrcode-stream-overlay {
-  width: 100%;
-  height: 100%;
+const wrapperStyle = ref<CSSProperties>({
```

Comment:
**Aside**: I think these could just be simple objects - no need to make them reactive